portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/mfa/MultiFactorAuthentication.testcase

```diff
@@ -572,6 +572,41 @@ definition {
 			userPassword = "test");
 	}
 
+	@description = "This is a use case for LPS-89360"
+	@priority = "4"
+	test ViewOrderOfOTPVerifier {
```

Comment:
Could you please add property test.name.skip.portal.instance = "ViewOrderOfOTPVerifier"; ?